src/components/molecules/UiScale/UiScale.vue

```diff
@@ -31,7 +32,8 @@
           v-model="scaleValue"
           :value="index"
           class="ui-scale__option"
-          :aria-labelledby="`scale-label-${index}`"
+          :aria-checked="scaleValue === index"
+          role="radio"
```

Comment:
? You get `<label role="radio" aria-checked="true"><input type="radio/></label>`. IMO it's not expected result. And I found that https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-checked

> Note: Where possible use an HTML [<input>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) element with type="checkbox" as this element has built in semantics and does not require ARIA attributes.